ape_starknet/transactions.py

```diff
@@ -156,8 +177,11 @@ class StarknetReceipt(ReceiptAPI, StarknetMixin):
 
     @property
     def ran_out_of_gas(self) -> bool:
-        # Errors elsewhere if we run out of gas.
-        return False
+        return self.max_fee == self.actual_fee
```

Comment:
Are we sure it cannot be higher than `max_fee`?

```suggestion
        return self.actual_fee >= self.max_fee
```